src/desktop/apps/fair/index.coffee

```diff
@@ -29,6 +29,7 @@ app.get '/:id/articles', getFairData, routes.fairArticles
 app.get '/:id/for-you', getFairData, routes.forYou
 app.get '/:id/search', getFairData, routes.search
 app.get '/:id/browse/show/:partner_id', getFairData, routes.showRedirect
+app.get '/:id/browse/artworks/artworks*', getFairData, routes.malformedFilterRedirect
```

Comment:
- Do we need `*` to match the query params? If not, we can omit it to make the route exact.
- We probably don't need to `getFairData` if we are going to redirect?

Reply:
Both were unnecessary. Removed them 🙌